content/graphics-switch-pop.md

```diff
@@ -63,7 +63,34 @@ GLX applications must be launched with this command to be rendered on the dGPU (
 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia <application>
 ```
 
-Hybrid graphics mode is available on 19.10 and later.
+Hybrid graphics mode is available on 19.10 and later. Applications can be launched using Hybrid graphics on a per-application basis and through inheritance of launch state from a game launcher.
+
+#### Hybrid Graphics Integration with Steam: GUI Method
+
+When Steam is favorited, can be launched using Discrete Graphics by right clicking on the Pop Dock and selecting the following option: 
```

Comment:
This should not be necessary. Steam's `.desktop` file contains `PrefersNonDefaultGPU=true`, which should make it use the dedicated GPU by default in Hybrid mode.